content/Api/Blip.Api.Template/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-alpine3.11 AS base
+FROM mcr.microsoft.com/dotnet/aspnet:5.0-alpine3.11 AS base
```

Comment:
This alpine image doesn't exist. Check the [Docker hub page](https://hub.docker.com/_/microsoft-dotnet-sdk/) to find one that suits. I would suggest using the 5.0-alpine3.12 version.